core/src/main/java/com/linecorp/armeria/client/endpoint/EmptyEndpointGroupException.java

```diff
@@ -32,12 +33,20 @@
      * Returns an {@link EmptyEndpointGroupException} which may be a singleton or a new instance, depending on
      * {@link Flags#verboseExceptionSampler()}'s decision.
      */
-    public static EmptyEndpointGroupException get() {
+    public static EmptyEndpointGroupException get(@Nullable EndpointGroup endpointGroup) {
```

Comment:
Although it is usually initiated by internally, this is a breaking change. Should we keep the original method and overload it?